packages/contractkit/src/identity/claims/claim.ts

```diff
@@ -103,6 +82,11 @@ export function hashOfClaim(claim: Claim) {
   return hashMessage(serializeClaim(claim))
 }
 
+export function hashOfClaims(claims: Claim[]) {
+  const hashes = claims.map(hashOfClaim)
+  return hashMessage(hashes.join(''))
```

Comment:
Do you get a predictable ordering here?

Reply:
If you are asking whether the `hashOfClaims` is yielding the same hash given an ordered `claims` parameter, then I believe so?

Reply:
oh yeah of course SG